rudder-agent/SOURCES/check-rudder-agent

```diff
@@ -70,26 +68,24 @@ clean_cf_lock_files() {
 check_and_fix_cfengine_processes() {
 
   # Detect the correct ps tool to use
-  # Supported tools are ps, vzps, and the rudder supplied vzps.py
-  PS=ps
-  VZPS=/bin/vzps
-  RUDDERPS=/opt/rudder/bin/vzps.py
-
-  # Detect if we are a VZ host
-  if [ -e "/proc/bc/0" ]; then
-    # Detect if vzps is present
-    if [ -e ${VZPS} ]; then
-      PS="${VZPS} -E 0"
-    else
-      PS="${RUDDERPS} -E 0"
+  ns=$(ps -h -o utsns --pid $$ 2>/dev/null)
+  if [ $? -eq 0 ]; then # we have namespaces
+    PS_COMMAND="ps -h -e -O utsns | grep -E '^[[:space:]]*[[:digit:]]*[[:space:]]+${ns}'"
```

Comment:
I'm asking because this command works on my Debian jessie (without any container management system installed, just because systemd is here seems to be enough) and in the case you have a ns enabled ps and OpenVZ installed, this modification would basically break OpenVZ support I guess (the vzps cases wouldn't even be evaluated)